contracts/Governance/GovernorAlpha.sol

```diff
@@ -150,23 +150,22 @@ contract GovernorAlpha {
         uint endBlock = add256(startBlock, votingPeriod());
 
         proposalCount++;
-        Proposal memory newProposal = Proposal({
-            id: proposalCount,
-            proposer: msg.sender,
-            eta: 0,
-            targets: targets,
-            values: values,
-            signatures: signatures,
-            calldatas: calldatas,
-            startBlock: startBlock,
-            endBlock: endBlock,
-            forVotes: 0,
-            againstVotes: 0,
-            canceled: false,
-            executed: false
-        });
-
-        proposals[newProposal.id] = newProposal;
+        uint proposalId = proposalCount;
+        Proposal storage newProposal = proposals[proposalId];
```

Comment:
Let's add a revert below this line if newProposal has any data set. This should never be the case, but if so, we don't want to overwrite anything.

Reply:
This comment applies to Governor Bravo as well.